File: milliseconds/golang/models/responses/millisecond_response.go
package modelresponses

type MillisecondResponse struct {
	Datetime            string `json:"datetime"`
	Millisecond         int64  `json:"millisecond"`
	Add1Hour            string `json:"add1Hour"`
	Add1HourMillisecond int64  `json:"add1HourMillisecond"`
}

func GetMillisecondResponse(datetime string, millisecond int64, add1Hour string, add1HourMillisecond int64) MillisecondResponse {
	return MillisecondResponse{
		Datetime:            datetime,
		Millisecond:         millisecond,
		Add1Hour:            add1Hour,
		Add1HourMillisecond: add1HourMillisecond,
	}
}

File: milliseconds/rust/src/models/responses/millisecond_response.rs
use serde::{Deserialize, Serialize};

#[derive(Debug, Deserialize, Serialize)]
pub struct MillisecondResponse {
    #[serde(rename = "datetime")]
    pub datetime: String,
    #[serde(rename = "millisecond")]
    pub millisecond: i64,
    #[serde(rename = "add1Hour")]
    pub add1_hour: String,
    #[serde(rename = "add1HourMillisecond")]
    pub add1_hour_millisecond: i64
}

impl MillisecondResponse {
    pub fn set_millisecond_response(datetime: String, millisecond: i64, add1_hour: String, add1_hour_millisecond: i64) -> MillisecondResponse {
        MillisecondResponse { datetime:datetime, millisecond: millisecond, add1_hour: add1_hour, add1_hour_millisecond: add1_hour_millisecond }
    }
}